### PR TITLE
VACMS-19821: Removes mental health number from Facility IEF

### PR DESCRIPTION
**Pull request by omahane**
## Description

Relates to #19821

## Testing done
Manually

## Screenshots
Screenshot 2024-11-06 at 5 00 31 PM

### Set up editor
1. Log in as an admin
- [x]  Assign the user QA Content Publisher to the "VA Boston health care" Section
- [x] Assign the following roles to QA Content Publisher
  - Content publisher
  - Content creator - VAMC

### Review the operating status
1. Log in as QA Content Publisher
2. Edit the [Operating status | VA Boston health care page](https://pr19822-2w3naylfv0vxes0tkokd8fmgnv3xsnqa.ci.cms.va.gov/node/27318/edit)
3. Scroll down to the Facility operating statuses
4. Click on the Edit of the first facility
- [x] Confirm that no "Mental health phone number" is shown

### Select Team for PR review

- [ ] `CMS Team`
- [ ] `Public websites`
- [x] `Facilities`
- [ ] `User support`
- [ ] `Accelerated Publishing`

### CODE REVIEW

**Review by jv-agile6 — APPROVED** [review]
Followed QA test steps to success. Reviewed the Facility operating statuses to determind there was no mental health phone field. Reviewed configuration changes, and approved.

File: config/sync/core.entity_form_display.node.health_care_local_facility.inline_entity_form.yml [hunk 1]
```diff
@@ -53,7 +51,6 @@ third_party_settings:
     group_editorial_workflow:
       children:
         - moderation_state
-        - revision_log
```
Comment (davidmpickett) [inline config/sync/core.entity_form_display.node.health_care_local_facility.inline_entity_form.yml:56]: Revision log should not be removed @omahane.

Comment (davidmpickett) [reply config/sync/core.entity_form_display.node.health_care_local_facility.inline_entity_form.yml:56]: Feel free to ignore this comment. IEF widget doesn't show this anyway, so it probably doesn't matter

**Comment by davidmpickett** [comment]
Works as intended. Approved.

**Review by jv-agile6 — APPROVED** [review]
Reviewed updated chages to core.entity_form_display.node.health_care_local_facility.inline_entity_form.yml and approved.

**Pull request merged**